cmd/server-main.go

```diff
@@ -187,12 +188,14 @@ func serverHandleEnvVars() {
 	// Handle common environment variables.
 	handleCommonEnvVars()
 
-	if serverRegion := env.Get("MINIO_REGION", ""); serverRegion != "" {
-		// region Envs are set globally.
-		globalIsEnvRegion = true
-		globalServerRegion = serverRegion
+	accessKey := env.Get(config.EnvAccessKey, auth.DefaultAccessKey)
+	secretKey := env.Get(config.EnvSecretKey, auth.DefaultSecretKey)
+	cred, err := auth.CreateCredentials(accessKey, secretKey)
+	if err != nil {
+		logger.Fatal(config.ErrInvalidCredentials(err),
+			"Unable to validate credentials inherited from the shell environment")
 	}
-
+	globalActiveCred = cred
```

Comment:
after migrating from old -> new config the AK,SK set as env var are not displayed correctly on the console. Instead old AK,SK from previous config.json are shown. globalActiveCred should not be overwritten while migrating if already set from environment.

Reply:
This is fixed already, I need to push new set of changes @poornas

Reply:
Unresolved since I had to bring this change back, I am doing a small change to preserve the existing deployments.

Reply:
This is fixed